extensions/rant_patrol.py

```diff
@@ -20,13 +22,21 @@ async def main(event: hikari.GuildMessageCreateEvent):
 
     messageContent = event.content
 
-    if not re.match("^(rant|vent): [a-zA-Z0-9_].*", messageContent):
-        response: str = (
-            f"> Your message must start with 'rant: ' OR 'vent: '. Try again or keep your complaints to yourself."
-        )
-        await event.message.respond(response, reply=True, user_mentions=True)
-    else:
-        return
+    """
+    "Rant: too many memes" ==> picked up by kitti
+    "rant: my website's domain got sniped" ==> all good (properly formatted)
+    "shut the fuck up jimmy"" ==> all good (standard comment)    
+    """
+
+    if re.match("^((anti)?(rant|vent):).", messageContent.lower().strip("-")):
+        if not re.match("^((anti-)?(rant|vent):).", messageContent):
+            response: str = (
+                f"> Your message must start with 'rant: ' OR 'vent: ' with the optional 'anti-' prefix. Try again or keep your complaints to yourself."
+            )
+            await event.message.respond(response, reply=True, user_mentions=True)
+        else:
+            return
```

Comment:
Rant: this is unnecessary

Reply:
Why?

Reply:
Both `return`s are! If the `else` branch is hit in either case, there is no further code in the function, and omitting a `return` is the same as having no return with no further code